docs/dashboard.md

```diff
@@ -1,17 +1,25 @@
-Gemdash includes a dashboard to interactively select your input data and experimental parameters. This makes it straightforward to explore your data without any coding experience required.
+GEMDAT includes a dashboard to interactively select your input data and experimental parameters. This makes it straightforward to explore your data without any coding experience required.
+
+The dashboard is available from: https://github.com/GEMDAT-repos/gemdat-dashboard
 
 To use the dashboard, first make sure you install all the dependencies:
 
 ```
-pip install gemdat[dashboard]
+pip install -r requirements.txt
```

Comment:
It would be nice if we can do `pip install gemdash-dashboard` (or `pip install gemdash`) Is there a reason you chose a "local" install method here?

Reply:
It would also mean that we get to keep the `gemdash` command, instead of having to `python gemdash.py`

Reply:
I thought about that, and came to the conclusion that I didn't want to go through the hoops of setting up all the boilerplate and upload to pypi etc at this point. Especially because I'm not certain how much effort we want to put into it. But that would definitely be nice to have.

Reply:
Yeah, and to keep the `gemdash` command on the dashboard side completely.